saleor/graphql/menu/types.py

```diff
@@ -54,6 +54,14 @@ class MenuItem(CountableDjangoObjectType):
         resolver=resolve_translation,
     )
 
+    sort_order = graphene.Field(
+        graphene.Int,
+        deprecation_reason=(
+            "Use the position in list instead and relative "
```

Comment:
We need to also say in which version it will be removed. I suggest 2.10 since 2.9 is about to be released.